### DISCUSSION TITLE
how do i erase this thing at the end of the hello world

### DISCUSSION THREAD

**Discussion by iJagerWolf** [microsoft/vscode-discussions#351] [Extension Development QnA] [answered]
hi there, just looking is there a way i can erase this "%" 
<img width="555" alt="Screenshot 2022-12-29 at 4 31 31 a m" src="https://user-images.githubusercontent.com/98992122/209938904-31dd6a80-07d2-410c-bc49-234261dde84d.png">

**Comment by burkeholland** [comment, marked as answer]
Thanks for your question! This question is specific to your terminal setup and is probably best answered via Stack Overflow. This discussion area is for extension related questions only.
